runtime/verbose/CMakeLists.txt

```diff
@@ -112,6 +109,9 @@ omr_add_exports(j9vrb
 	J9VMDllMain
 )
 
+target_enable_ddr(j9vrb GLOB_HEADERS)
```

Comment:
@dnakamura Can you comment on whether `GLOB_HEADERS` is the right choice here?

Reply:
Yes. Its generally the easiest choice to avoid having to list all the headers (and open yourself to the possibility of forgetting when new headers are added in the future)